src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -169,5 +170,45 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
 
             return bestItems.ToImmutableAndFree();
         }
+
+        internal static async Task<CompletionDescription> GetDescriptionAsync(
+            CompletionService service, CompletionItem item, CancellationToken cancellationToken = default)
```

Comment:
This looks thoroughly broken to have a static method on a service take an instance of the service. That should be an instance method IMO.